### PR TITLE
Fixed descriptions

### PR DESCRIPTION
**Pull request by eldadfux**

### CODE REVIEW

File: src/SDK/Language/PHP.php [hunk 1]
```diff
@@ -10,8 +10,8 @@ class PHP extends Language {
      * @var array
      */
     protected $params = [
-        'composerVendor' => 'vendorName',
-        'composerPackage' => 'packageName',
+        'composerVendor' => 'vendor-name',
```
Comment (lohanidamodar) [inline src/SDK/Language/PHP.php:13]: Not sure why this change was made?

**Review by lohanidamodar — APPROVED** [review]
LGTM.

File: src/SDK/Language/PHP.php (hunk 1 above)
Comment (eldadfux) [reply src/SDK/Language/PHP.php:13]: Composer seem to have issue with uppercase letters in these values.
